src/lib/legacy.ts

```diff
@@ -4676,6 +4676,15 @@ const fromZigbee1 = {
             }
             if (dp === dataPoints.hochActivePower) {
                 result.power = (value[2] | value[1] << 8) / 10;
+                if (value.length > 3) {
+                    result.power_1 = (value[5] | value[4] << 8) / 10;
```

Comment:
can you rename these to e.g. `power_l1`? (add `l`)

Reply:
sure. done